index.html

```diff
@@ -38,6 +39,204 @@ <h2 id="job-title">I am a Software Developer</h2>
+            <section class="about-me">
+                <div id="about-div">
+                    <h2 id="about-me-title">About Me</h2>
+                </div>
+                <div id="about-me-description-div">
+                    <p id="about-me-description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <a href=""><button id="get-resume-button">Get My Resume</button></a>
+                </div>
+                <p id="line3"></p>
+                <div class="languages-div">
+                    <div class="languages">
+                        <ul id="languages">Language
+                            <li id="individual-languages">JavaSript</li>
+                            <li id="individual-languages">Ruby</li>
+                            <li id="individual-languages">Html</li>
+                            <li id="individual-languages">Css</li>
+                        </ul>
+                    </div>
+                    <div class="frameworks-div">
+                        <ul id="languages">Frameworks
+                            <div class="mini-language-div">
+                                <li id="individual-frameworks">Bootstrap</li>
+                                <li id="individual-frameworks">Ruby on Rails</li>
+                                <li id="individual-frameworks">RSPec</li>
+                                <li  id="individual-frameworks">CapyBara</li>
+                                <li id="individual-frameworks">Selenium</li>
+                            </div>
+                            
+                        </ul>
+                    </div>
+                    <div class="skills-div">
+                        <ul class="skills-skills" id="languages">Skills
+                            <li id="individual-skills">Codekit</li>
+                            <li id="individual-skills">GitHub</li>
+                            <li id="individual-skills">Codepen</li>
+                            <li id="individual-skills">Gitlab</li>
+                            <li id="individual-skills">Terminal</li>
+                        </ul>
+                    </div>
+                </div>
+            </section>
+            <section class="footnote">
+                <div id="brief-note-div">
+                    <h3 id="brief-note">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+                </div>
+                <div class="get-in-touch">
+                    <input class="get-in-touch-individuals" type="text" placeholder="Full name">
+                    <input type="text" placeholder="Email address">
+                    <input type="text" placeholder="Enter text here">
+                    <a href=""><button id="get-in-touch-button">Get in touch</button></a>
```

Comment:
Kindly, the form fields should be wrapped by the "form" label

Reply:
Kindly, this suggestion that I did the last revision did not do it, I recommend doing it.